### ISSUE TITLE
Main menu label support

### ISSUE THREAD

**Issue by surivrox** [avanzu/AdminThemeBundle#63]
How can I add a header to the left menu? Is it supported?

``` html
<ul class="sidebar-menu">
<li class="header">...</li>
</ul>
```

**Comment by surivrox** [comment]
This pull request resolves above problem https://github.com/avanzu/AdminThemeBundle/pull/68

**Comment by shakaran** [comment]
@chteuchteu This issue could be closed, since PR #68 resolve the problem

**Comment by shakaran** [comment]
Closing since resolved
